content/pages/download.md

```diff
@@ -158,6 +158,17 @@ versions:
 
       **Development snapshots are not intended for live use!** Expect crashes and make sure to back up your Mixxx settings and library before upgrading as explained in the Testing wiki page.
     downloads:
+      - slug: ubuntu
+        os: Ubuntu 18.04 "Bionic Beaver" or later
```

Comment:
IIRC Qt >=5. 12 is a requirement for 2.4.

```suggestion
        os: Ubuntu 20.04 "Focal Fossa" or later
```

Reply:
Not yet. A Bionic 2.4 package is available.

Reply:
Maybe it still builds, but build with Qt < 5.12 will lack the ES6/ES7 QJSEngine feature set, which means the controller mappings using these features will not work. 

Also, since we don't have a built-in updater, someone who installs this via PPA will suddenly receive updates without noticing when support for Bionic is dropped.

Reply:
OK